recipe/meta.yaml

```diff
@@ -55,6 +54,17 @@ test:
   commands:
     - swipl -g "writeln('hello world')." -t 'halt.'
 
+outputs:
+  - name: {{ name|lower }}_openssl_3
```

Comment:
I don't recommend doing this with multiple outputs, or hard-pinning, but rather _variants_ that use `conda-build-config.yaml`: here's how [hdf5](https://github.com/conda-forge/hdf5-feedstock/blob/6ca06ec7334069620ef5dbc70368e0815fd3bcdd/recipe/conda_build_config.yaml#L7) does it.


```yaml
# recipe/conda-build-config.yaml
openssl:
- '3'
- '1.1.1'
```

```yaml
# recipe/conda-build-config.yaml
host:
  - openssl
```

This will:
- build multiple excursions in parallel 
- let the solver do more of the work than a user having to dig around for a specific transient dependency version
- let the bot know what's happening, e.g. recompile against a new openssl `1.x` instead of hard-pininng

Reply:
@bollwyvl perfect, I was hoping there was a better way :) 

I'll update the PR.

Reply:
@bollwyvl look good to you?